content/contributing/documentation.rst

```diff
@@ -312,19 +312,22 @@ with :abbr:`RST (reStructuredText)` syntax (see :doc:`documentation/rst_cheat_sh
 guidelines.
 
 .. important::
-   If your changes include the addition of a new image, for instance :file:`my_image.png`, proceed
-   as follows:
+   - If your changes include the addition of a new image, for instance :file:`my_image.png`, proceed
```

Comment:
media files use hyphens
https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files
```suggestion
   - If your changes include the addition of a new image, for instance :file:`my-image.png`, proceed
```

Reply:
and following examples too